Source/Fuse.ImageTools/Tests/ImageTools.Test.uno

```diff
@@ -10,49 +10,31 @@ namespace Fuse.ImageTools.Test
 	{
 		private Object _image = null;
 
-
 		[Test]
+		[Ignore("Only supported on dotnet", "!DOTNET")]
```

Comment:
How can this be? `ImageFromBase64` does `closure.Reject("Unsupported platform");` when we're not on Android nor iOS, as far as I can tell...